src/page-objects/storefront/CheckoutCart.ts

```diff
@@ -26,6 +29,9 @@ export class CheckoutCart implements PageObject {
         this.cartLineItemImages = page.locator('.line-item-img-link');
         this.unitPriceInfo = page.locator('.line-item-unit-price-value');
         this.cartQuantityNumber = page.locator('input[name="quantity"]');
+        this.productNameLabel = page.locator('.line-item-label');
+        this.productNumberLabel = page.locator('.line-item-product-number');
+        this.productDeliveryDateLabel = page.locator('.line-item-delivery-date');
```

Comment:
For visual tests we still need to generate a product right? So wouldn't we have access to that information and could define (at least the first two locators) dynamically like so:

```suggestion
       public readonly productNameLabel:(productName: string) => Locator;
       public readonly productNumberLabel:(productNumber: string) => Locator;

        this.productNameLabel = (productName: string) => page.getByRole('link', { name: productName });
        this.productNumberLabel = (productNumber: string) => page.getByText(productNumber);
```
For the DeliveryDate we could at least partially match against snippets. This would have the advantage of not only checking if the locator exists but also that it matches expected content.